es-ds-components/components/es-progress.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+    <ProgressBar
+        aria-describedby="es-progress"
```

Comment:
This aria-describedby doesn't seem to be referring to anything. It's in the old one as well.. In any case I think we can remove it, and downstream users should apply `aria-labelledby`, `aria-label`, or `aria-desribedby` to `<es-progress>` which will automatically pass through to PrimeVue `<progress-bar>`.